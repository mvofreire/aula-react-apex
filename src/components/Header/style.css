.header{
  background: #f0f0f0;
  text-align: left;
  position: relative;
  display: block;
  padding: 20px;
  font-size: 20px;
  border-bottom: 1px solid #ddd;
}