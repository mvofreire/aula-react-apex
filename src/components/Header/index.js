import React from 'react'
import { useTaskContext } from '../../context/task.context'
import "./style.css"

function Header() {
  const { tasks } = useTaskContext()
  return (
    <div className='header'>
      Minhas Tarefas (Quantidade: {tasks.length})
    </div>
  )
}

export default Header