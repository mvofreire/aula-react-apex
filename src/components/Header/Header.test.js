import { render } from '@testing-library/react';
import Header from '.';

describe('Header', () => {
  test('Should render in screen', () => {
    const { container } = render(<Header />)
    expect(container).toBeInTheDocument()
  })

  test('Should match with snapshot', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot('default')
  })
})