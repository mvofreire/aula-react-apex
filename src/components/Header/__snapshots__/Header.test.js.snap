// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should match with snapshot: default 1`] = `
<div>
  <div
    class="header"
  >
    Minhas Tarefas
  </div>
</div>
`;
